lib/oidcc/token.ex

```diff
@@ -353,6 +353,31 @@ defmodule Oidcc.Token do
       |> :oidcc_token.client_credentials(opts)
       |> normalize_token_response()
 
+  @doc """
+  Generate a map of authorization headers to use when using the given
+  `%Oidcc.Token.Access{}` to access an API endpoint.
+  """
+  @doc since: "3.2.0"
+  @spec authorization_headers(
```

Comment:
Should this function be on `Oidcc.Token.Access` since we have a separate module in elixir for the struct?